lib/aerospike/value/value.rb

```diff
@@ -26,10 +26,8 @@ class Value #:nodoc:
 
     def self.of(value)
       case value
-      when nil
```

Comment:
we compare value class here one by one, so for each value we first compared to `NilClass`. I think we shouldn't expect a lot of these, so I moved it to the bottom